### DISCUSSION TITLE
请问现在highlight.js仓库base16文件夹下高亮风格如何使用

### DISCUSSION THREAD

**Discussion by Wkingxc** [ppoffice/hexo-theme-icarus#1202] [Q&A] [answered]
求教，感谢！

**Comment by ppoffice** [comment, marked as answer]
1. https://github.com/ppoffice/hexo-theme-icarus/blob/fabb67bb6099b3bc409a5c033136f6ef97283754/layout/common/head.jsx#L169 这一行`9.12.0`改为`11.7.0`。
2. `_config.yml`或者`_config.icarus.yml`中`article.highlight.theme`设置为`base16/dracula`或者其他`base16/*`。

**Reply by Wkingxc** [reply]
![image](https://user-images.githubusercontent.com/73375688/232714176-f5439a8d-2e24-4da8-97cd-71c463bffbd8.png)
大佬想请问下，为什么只要是用dark风格的高亮，一些函数代码颜色都是非常暗灰，基本看不清，例如onedark之类的
![image](https://user-images.githubusercontent.com/73375688/232714776-d3ede705-362f-494c-94c7-fc88e02cc243.png)

**Reply by ppoffice** [reply]
再将这几行：
https://github.com/ppoffice/hexo-theme-icarus/blob/fabb67bb6099b3bc409a5c033136f6ef97283754/source/js/main.js#L65-L68
替换成下面：
```js
for (const cls of classes) {
    $(this).addClass('hljs-' + cls);
    $(this).removeClass(cls);
}
```

**Reply by Wkingxc** [reply]
非常感谢！
